index.html

```diff
@@ -122,49 +123,49 @@ <h1 tabindex="0">I'm Racha, <br> Glad to see you!</h1>
         />
         </div>
         <div class="project-content">
-          <h1 class="project-heading">Tonic</h1>
+          <h1 tabindex="0" class="project-heading">Tonic</h1>
```

Comment:
- [ ] please kindly ensure that  The heading hierarchy is meaningful. Ideally the page starts with an "h1"`(I'm Racha, <br> Glad to see you!)`. i can see you have many `h1` heading tags, you can change them to `h2` so as to give them a meaningful hierarchy. [see more here](https://www.w3.org/WAI/test-evaluate/preliminary/#headings)